src/components/search/SearchComponent.vue

```diff
@@ -40,7 +40,13 @@ export default Vue.extend({
   },
   methods: {
     handleSearchValueChange: debounce(function () {
-      this.$emit('searchChanged', this.searchValue)
+      if (!this.searchValue || this.searchValue.length >= 3) {
+        this.lastSearched = this.searchTerm
+        this.$emit('searchChanged', this.searchValue)
+      } else if (this.searchValue.length > 0 && this.searchValue.length < 3) {
+        this.lastSearched = ''
+        this.$emit('searchChanged', '')
```

Comment:
The update on 1 and 2 letters gives the impression that you are doing a search. ( requests are being made, data is loaded, spinners are spinning )
And this will be a 'broken' search.

Maybe only search if lastSearched differs from searchValue?